### PR TITLE
Port to new library support dotnet core

### PR DESCRIPTION
**Pull request by KhuongNtrd**
For issues #221

### CODE REVIEW

File: SendGrid/src/SendGrid.Sample/Program.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SendGrid.Sample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
```
Comment (KhuongNtrd) [inline SendGrid/src/SendGrid.Sample/Program.cs]: oops, sorry, just delete Program.cs to run example

**Comment by KhuongNtrd** [comment]
@thinkingserious , check commit https://github.com/sendgrid/sendgrid-csharp/pull/320/commits/3282e5cae21c7494886b1d7c45fb4c62b29e8002

File: SendGrid/src/SendGrid.Sample/Example.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ private static void Main()
 
         private static async Task HelloEmail()
         {
-            String apiKey = Environment.GetEnvironmentVariable("SENDGRID_APIKEY", EnvironmentVariableTarget.User);
+            String apiKey = Environment.GetEnvironmentVariable("SENDGRID_APIKEY");
```
Comment (KhuongNtrd) [inline SendGrid/src/SendGrid.Sample/Example.cs]: 'cause that sample add is dotnet portable app, not have specific runtime, so you cound run with Environment Variable by using "SET" and "dotnet run" command. In last commit (https://github.com/sendgrid/sendgrid-csharp/pull/320/commits/3282e5cae21c7494886b1d7c45fb4c62b29e8002), I have change sample app to Standard .net Console app in .net 4.5.2.

**Comment by KhuongNtrd** [comment]
``

File: SendGrid/src/SendGrid/project.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "System.Collections.Specialized": "4.0.1",
+        "System.Dynamic.Runtime": "4.0.11"
+      }
+    }
+  },
+  "scripts": {
+    "postcompile": [
+      "dotnet pack --no-build --configuration %compile:Configuration%"
+    ]
+  }
```
Comment (KhuongNtrd) [inline SendGrid/src/SendGrid/project.json]: here your package description, the *.nupkg file just place om bin/Release | Debug folder

File: SendGrid/src/SendGrid/project.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{
+  "version": "1.0.1-beta-*",
```
Comment (KhuongNtrd) [inline SendGrid/src/SendGrid/project.json]: here your package description file, the *.nupkg file just placed in bin/Release | Debug folder

File: SendGrid/src/SendGrid/project.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/sendgrid/sendgrid-csharp"
+    },
+    "tags": [
+      "sendgrid",
+      "email",
+      "microsoft",
+      "azure"
+    ]
```
Comment (KhuongNtrd) [inline SendGrid/src/SendGrid/project.json]: here your package description file, the *.nupkg file just placed in bin/Release | Debug folder

**Comment by thinkingserious** [comment]
Hello @KhuongNtrd,

Thanks for the additional updates!

1. It looks like we still have a few conflicts, please see above.
2. I do not have a CLA on file for you, could you please take a look and get a signed copy over to us? https://github.com/sendgrid/sendgrid-csharp/blob/master/CONTRIBUTING.md#cla

Thanks!

**Comment by thinkingserious** [comment]
Hello @KhuongNtrd,

I have added the .NET Core support to the v9beta branch: https://github.com/sendgrid/sendgrid-csharp/tree/v9beta

When you have a chance can you take a look? Specifically, I'm having trouble getting the tests to work in Travis.

Thanks!

**Pull request closed without merging**